src/interactionManager/types.ts

```diff
@@ -162,7 +162,7 @@ export type EncryptionRequest = RequestMessage<{
 }>
 export type EncryptionResponse = ResponseMessage<Base64String>
 export type SigningRequest = RequestMessage<{
-  target: DidDocKeyId
+  target?: DidDocKeyId
```

Comment:
What issue does this address? (Or were the types incorrect / outdated?)

Reply:
the types previously were not enforced strictly (because it was just creating a message and returning a string), so this was an inaccurate type which, when the types became enforced, would cause issues